src/providers/cortex/trulens/providers/cortex/endpoint.py

```diff
@@ -75,21 +132,22 @@ def handle_wrapped_call(
     ) -> None:
         counted_something = False
 
-        if isinstance(
-            response, DataFrame
-        ):  # response is a snowflake dataframe instance
+        # response is a snowflake dataframe instance or a list if the response is from cursor.fetchall()
+        if isinstance(response, DataFrame):
             response: dict = json.loads(response.collect()[0][0])
```

Comment:
I realize you didn't do this at all, but for L137 and L139 we seem to be assuming something about the response shape. Can we just raise errors if it's not that shape in case things change?

Reply:
done